### DISCUSSION TITLE
Support for Local HW Audio Jack

### DISCUSSION THREAD

**Discussion by DellanX** [music-assistant/support#625] [Q&A] [answered]
It'd be nice to have the native device's Audio Jack usable by this integration as well.
For example, I have a Raspberry PI running home assistant plugged into a speaker system. It'd be nice to be able to use Music Assistant to play on such device.

**Comment by marcelveldt** [comment, marked as answer]
That will be out of scope for Music Assistant integration but very easy to create as Home Assistant addon.
Just create a HA addon that wraps squeezelite and you're all set.

1) Run squeezelite on your HA hardware
2) Add the 'slimproto' integration to HA
3) enjoy

As a test you can even run it inside the ssh addon which I believe has access to the host audio.
squeezelite may even be available on the package manager so it might just be as simple as this:
- apk add squeezelite 
- run squeezelite

**Reply by Dodge883** [reply]
it's very easy
add vlc addon
and you're ready to go

**Reply by erkr** [reply]
> it's very easy
> 
> add vlc addon
> 
> and you're ready to go

Yep 👍 but then also add the vlc telnet integration
